### PR TITLE
Make the README's Discord badge fancier

### PR DESCRIPTION
**Pull request by dominiklohmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +26,12 @@ Check out [our documentation](https://docs.tenzir.com/) where your find
 tutorials that walk your through the first steps, how-to guides to solve a
 specific problem, explanations of key concepts, and an in-depth reference of the
 nitty-gritty technical details.
+
+## Community
+
+<a href="https://discord.gg/xqbDgVTCxZ" alt="Tenzir Discord community">
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://invidget.switchblade.xyz/xqbDgVTCxZ">
+  <img alt="Tenzir Discord community" src="https://invidget.switchblade.xyz/xqbDgVTCxZ?theme=light">
```
Comment (mavam) [inline README.md:35]: Is the hard coding of theme `light` on purpose here?

File: README.md [hunk 2]
```diff
@@ -34,3 +26,12 @@ Check out [our documentation](https://docs.tenzir.com/) where your find
 tutorials that walk your through the first steps, how-to guides to solve a
 specific problem, explanations of key concepts, and an in-depth reference of the
 nitty-gritty technical details.
+
+## Community
+
```
Comment (mavam) [inline README.md:31]:
```suggestion

Got questions? We're here to help. Join our friendly community Discord server where you'll find a thriving group of enthusiasts that love the intersection of data infrastructure and security operations.

```

**Review by mavam — APPROVED** [review]
Neat.

File: README.md (hunk 1 above)
Comment (dominiklohmann) [reply README.md:35]: It's both light and dark, this is the snippet recommended from upstream. See the `<source ...>` tag right above, which defines the dark mode image.

**Pull request merged**